packages/Liquid-Tests.package/LiquidLintTests.class/instance/testNoNonBlocksInSpecialMessages.st

```diff
@@ -0,0 +1,4 @@
+misc
+testNoNonBlocksInSpecialMessages
+
+	self assertAllZero: (self lint: #smallLintIfTrueBlocks)
```

Comment:
[Dust] Is this correct? `NonBlocksInSpecialMessages` means the same as `IfTrueBlocks`?